### ISSUE TITLE
black screen issue

### ISSUE THREAD

**Issue by dj5928** [valadaa48/retroroller#203]
Sometimes during gameplay
The screen turns off for no reason.
The sound and game will continue and only the screen will turn black.
The screen is displayed again only after entering the sleep mode.

**Comment by valadaa48** [comment]
This is on OGS?

**Comment by valadaa48** [comment]
duplicate of #197
